Mono.Debugging/Mono.Debugging.Client/DebuggerStatistics.cs

```diff
@@ -120,10 +126,18 @@ public class DebuggerTimer : IDisposable
 		readonly DebuggerStatistics stats;
 		readonly Stopwatch stopwatch;
 
-		public DebuggerTimer (DebuggerStatistics stats)
+		static readonly TraceListener traceListener = new TraceSource (nameof (DebuggerTimer)).Listeners.OfType<TraceListener> ().FirstOrDefault (l => l.Name == nameof (DebuggerTimer));
```

Comment:
Can we comment how to use this? I am assuming you need to setup a trace listener in VS Mac?